recipes/mongo-c-driver/all/conanfile.py

```diff
@@ -93,6 +93,8 @@ def build_requirements(self):
             if not self.conf.get("tools.gnu:pkg_config", check_type=str):
                 self.tool_requires("pkgconf/2.1.0")
 
+        self.tool_requires("cmake/[>=3.15.7]")
```

Comment:
Please remove unless a version higher than 3.15 is required, thanks!
(see https://github.com/conan-io/conan-center-index/issues/27092#issuecomment-2772403786 for additional context)